addons/hr/models/hr_employee.py

```diff
@@ -364,9 +364,9 @@ def get_formview_action(self, access_uid=None):
         if access_uid:
             self_sudo = self.with_user(access_uid)
```

Comment:
if we are in that condition, `self_sudo` does not have `has_group` method and so the bug will be always there.